ts/features/bonus/bpd/components/BpdTransactionSummaryComponent.tsx

```diff
@@ -35,18 +35,22 @@ const styles = StyleSheet.create({
   },
   readMore: { marginLeft: 31, marginBottom: 24 }
 });
+
+const readMoreLink = "https://io.italia.it/cashback/acquirer/";
+const SHOW_CTA_DELAY = 500;
+
 const CSS_STYLE = `
 body {
   font-size: 16;
   color: ${IOColors.black}
 }
 `;
 
-const readMoreLink = "https://io.italia.it/cashback/acquirer/";
-export const bottomSheetBpdTransactionsBody = () => {
+export const BottomSheetBpdTransactionsBody: React.FunctionComponent = () => {
   const [CTAVisibility, setCTAVisibility] = React.useState(false);
 
-  const setCTAVisible = () => setCTAVisibility(true);
+  const setCTAVisible = () =>
+    setTimeout(() => setCTAVisibility(true), SHOW_CTA_DELAY);
```

Comment:
```suggestion
    setCTAVisibility(true);
```

We can avoid the usage of a `setTimeout` here, I tested the solution with the direct call of a setState and it actually works

Reply:
@thisisjp If we remove `setTimeout` the CTA will actually be visible a fraction of a second _before_ the content appears, but we get better code readibility, is this ok with you?

Reply:
no problem at all, thanks ;)